io_scene_nif/modules/property/shader/shader_import.py

```diff
@@ -181,3 +181,32 @@ def find_bsshaderproperty(self, n_block):
             else:
                 bsshaderproperty = self.bsShaderProperty1st
         return bsshaderproperty
+
+    def import_uv_offset(b_mat, shader_prop):
+        for texture_slot in b_mat.texture_slots:
+            if texture_slot:
+                texture_slot.offset.x = shader_prop.uv_offset.u
+                texture_slot.offset.y = shader_prop.uv_offset.v
+
+    def import_uv_scale(b_mat, shader_prop):
+        for texture_slot in b_mat.texture_slots:
+            if texture_slot:
+                texture_slot.scale.x = shader_prop.uv_scale.u
+                texture_slot.scale.y = shader_prop.uv_scale.v
+
+    def import_clamp(b_mat, shader_prop):
+        clamp = shader_prop.texture_clamp_mode
```

Comment:
This looks like a bitfield, could simplify the if cases below by checking the relevant bits
0 = 00000000
1 = 00010000
2 = 00100000
3 = 00110000

Reply:
Yeah haven't looked at the shader code, just wanted to pull it out from the older style texture code. 
Good suggestion